templates/Default/engine/Default/planet_construction.php

```diff
@@ -1,4 +1,4 @@
-<p>You are currently building: <?php
+<p><span name="currently_building">You are currently building: <?php
```

Comment:
Can you remove this span (or fully convert the file to specific refresh areas) - as it stands it just adds more html without changing anything (at least I don't think it changes anything)